continuous-integration/linux/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR=$(realpath "$SCRIPT_DIR/../..")
+
+"$SCRIPT_DIR/lint.sh"
```

Comment:
Missing || exit, but I propose to keep this out of this file. My proposal is to have 4 top level scripts called
setup.py
build.py
test.sh
lint.sh